### PR TITLE
MKS 22 Create new WEB API project

### PR DESCRIPTION
**Pull request by tann96**
MKS Create ASP.NET Core 2.2 WEB API project #22

### CODE REVIEW

**Review by TsisinskyjV — APPROVED** [review]

File: Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+﻿using System;
```
Comment (bvzayats) [inline Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs]: Please remove unused namespaces

File: Tetiana_Sulyma/Tetiana_Test/Program.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
```
Comment (bvzayats) [inline Tetiana_Sulyma/Tetiana_Test/Program.cs]: Please remove unused namespaces

File: Tetiana_Sulyma/Tetiana_Test/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
+﻿using System;
```
Comment (bvzayats) [inline Tetiana_Sulyma/Tetiana_Test/Startup.cs]: Please remove unused namespaces in the whole system

**Review by bvzayats — COMMENTED** [review]
Don't use '_' character in the branch naming. Please fix it.

File: Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs (hunk 1 above)
Comment (tann96) [reply Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs]: Done

**Comment by Magfiozy** [comment]
Merge development into this branch

File: Tetiana_Sulyma/.gitignore [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.vs/
```
Comment (bvzayats) [inline Tetiana_Sulyma/.gitignore]: Remove this file

File: Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs [hunk 5]
```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
```
Comment (bvzayats) [inline Tetiana_Sulyma/Tetiana_Test/Controllers/ValuesController.cs]: Removed unused namespaces.

**Pull request closed without merging**